c2corg_ui/static/js/utils.js

```diff
@@ -2,6 +2,26 @@ goog.provide('app.utils');
 
 
 /**
+ * @param {string} type Short document type code.
+ * @return {string} Full document type name.
+ */
+app.utils.getDoctype = function(type) {
+  // TODO add other types
+  switch (type) {
+    case 'w':
+      return 'waypoints';
+    case 'r':
+      return 'routes';
+    case 'o':
+      return 'outings';
+    case 'i':
+      return 'images';
+    default:
+      return '';
```

Comment:
You mean:
```
    default:
      goog.assert.fail('Unrecognized type: ' + type);
      return '';
```
(not sure the final "return" is needed?)?

I have tried to add ``"goog.asserts.ENABLE_ASSERTS=true",`` to https://github.com/c2corg/v6_ui/blob/master/build.json#L32-L39
(as found in the doc of ``goog.assert.fail`` http://google.github.io/closure-library/api/namespace_goog_asserts.html) but I keep having the following build error:
> ERR! compile /home/alex/Desktop/v6_ui/c2corg_ui/static/js/utils.js:20: ERROR - Property assert never defined on goog
ERR! compile       goog.assert.fail('Unrecognized type: ' + type);